packages/jaeger-ui/src/components/common/copy-icon.js

```diff
@@ -0,0 +1,69 @@
+// @flow
```

Comment:
Because this is has a component as the default export, can you name the file `CopyIcon.js`?

Reply:
renamed